code/policies/BackwardsCompatibleCachingPolicy.php

```diff
@@ -37,7 +37,7 @@ class BackwardsCompatibleCachingPolicy extends HTTP implements ControllerPolicy
      *	user-agent clustering in some way, otherwise this will be an equivalent to disabling caching as there
      *	is a lot of different UAs in the wild.
      */
-    public $vary = 'Cookie, X-Forwarded-Protocol, User-Agent, Accept';
+    public $vary = 'X-Requested-With';
```

Comment:
Note that the `2.x` release line has removed the `BackwardsCompatibleCachingPolicy` via https://github.com/silverstripe/silverstripe-controllerpolicy/commit/70bf83211b4d800034492d3fd75ed0827f07a93e. This has been tagged as 2.0.0, which is part of the CWP 2.0.0 release.

@robbieaverill Could you clarify why it was removed? Not saying it shouldn't, just trying to get the full story.

Reply:
From memory it did a very similar thing to one of the other policies so I would've removed it to trim tech debt

Reply:
Note that his is is a bit of a misnomer now, since https://github.com/silverstripe/silverstripe-framework/pull/8242 removed `X-Requested-With` vary header from 3.7 (to be released as 3.7.2). I'm happy to keep it, since we discourage people using the module anyway.